Package.swift

```diff
@@ -1,13 +1,13 @@
-// swift-tools-version: 5.8
+// swift-tools-version: 5.9
 // The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
     name: "CubeFoundation",
     platforms: [
-        .iOS(.v15),
-        .macOS(.v10_15)
+        .iOS(.v16),
+        .macOS(.v11)
```

Comment:
Out of interest, do we have to specify mac? 🤔

Reply:
Don't have too, it's good to support as much as possible but admittedly this framework uses UIKit in many places so many it's easier to just not support mac